src/dreyfus_index.erl

```diff
@@ -222,6 +243,35 @@ code_change(_OldVsn, State, _Extra) ->
 
 % private functions.
 
+get_index_with_sig(#doc{body={Fields}}=Doc, Sig) ->
```

Comment:
This function only looks for an index of given signature in a single design document. A search index might be referred to by multiple design documents.